core/extraction/towns_fund_round_one.py

```diff
@@ -1370,24 +1373,21 @@ def extract_submission_refs(df_dict: pd.DataFrame) -> pd.DataFrame:
     return df_dict
 
 
-def extract_organisation_refs(df_dict: pd.DataFrame, round_1_data: pd.DataFrame) -> pd.DataFrame:
+def extract_organisation_refs(df_dict: dict[str, pd.DataFrame]) -> dict[str, pd.DataFrame]:
     """
-    Extracts submission IDs from the DataFrame dictionary and populates the 'Organisation_Ref' table
+    Populates the 'Organisation_Ref' table
```

Comment:
Some of these docstrings don't meet PEP guidance, ie they are supposed to be in imperative tense, line layout etc. there is actually a tool to check them called pydocstyle, we could potentially add it to the pre-commit at some point to catch these, but you can run it locally if you want to check which ones are wrong. 

No worries if you're too busy though, as it will pick up a lot of problems including ones not part of this PR.